2/contrib/s2i/run

```diff
@@ -105,6 +105,28 @@ function migrate_slave_configuration() {
   sed -i "s#<image>registry.access.redhat.com/openshift3/jenkins-slave-nodejs-rhel7\(:.*\)\{0,1\}</image>#<image>registry.access.redhat.com/openshift3/jenkins-slave-nodejs-rhel7:${JENKINS_SLAVE_IMAGE_TAG}</image>#" /var/lib/jenkins/config.xml
 }
 
+function force_copy_plugins() {
+   # if rhel
+   if [ "$(ls /usr/lib/jenkins/*.hpi 2>/dev/null)" ]; then
+      echo "This openshift jenkins image is starting in a deployment where the image version"
+      echo "was not recoreded.  A one time replacement of all the plugins from the image to the"
+      echo "jenkins plugin directory will occur."
+      echo ""
+      ls -la /usr/lib/jenkins
```

Comment:
is this for debug or are you intending to leave it in?

Reply:
nope suppose to remove ... part of next push